test/src/main/java/org/apache/accumulo/test/ManagerRepairsDualAssignmentIT.java

```diff
@@ -150,12 +164,24 @@ public void test() throws Exception {
     }
   }
 
+  private int getZkDeadCount(ZooReader zooReader, final InstanceId iid) {
+    String tPath = Constants.ZROOT + "/" + iid + Constants.ZDEADTSERVERS;
+    try {
+      int count = zooReader.getChildren(tPath).size();
+      log.debug("Current dead server count: {}", count);
+      return count;
+    } catch (Exception ex) {
+      throw new IllegalStateException(
+          "Failed to read the number of dead tservers reported in ZooKeeper", ex);
+    }
```

Comment:
You don't need to catch and rethrow the exception in the test code... just let it be thrown out of the test method and JUnit will ensure we see the stack trace. It doesn't need to be converted to a RTE.

Reply:
Changed in 4994f630bf